app/templates/closed-wiki.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="wiki-container closed-wiki">
+  {{wikia-ui-components/wiki-page-header
+    title="This wiki has been closed or disabled"
+  }}
+  {{#wds-button
```

Comment:
<WdsButton @href="?useskin=oasis" @class="closed-wiki__button" />

would be nicer with splattributes but DS does not support it yet